### PR TITLE
[DONE] Misc. 7.62x39mm fixes

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
Fixed the reloaded 7.62x39mm M43 description by adding an edited version of the base cartridge's description with all references to 57-N-231 and steel core removed. Also increased 762_m43 spawns to 20. The 7.62x39 could get an even greater buff to spawns, since large 500-round spamcans filled with 7.62x39 are sold at some American sporting goods stores alongside .223.

### CODE REVIEW

File: data/json/items/ammo/762.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     "copy-from": "762_m43",
     "type": "AMMO",
     "name": "reloaded 7.62x39mm M43",
+    "description": "7.62x39mm ammunition with 121.9gr bullets.  Developed in WW2 by the Soviet Union the 7.62x39mm round rapidly became extremely popular all over the world.  The bullet has poor wounding potential due to its stability, only begining to yaw after 26cm.",
```
Comment (Cyrano7) [inline data/json/items/ammo/762.json]: "...only begining to yaw after 26cm." That should be **beginning** with two 'n's.

Comment (Cyrano7) [reply data/json/items/ammo/762.json]: Could you fix the original 762_m43 entry also while your at it.

Comment (BorkBorkGoesTheCode) [reply data/json/items/ammo/762.json]: Fixed

**Comment by BorkBorkGoesTheCode** [comment]
Jenkins, rebuild please.

**Pull request merged**